extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -61,7 +61,10 @@ class AmpInstagram extends AMP.BaseElement {
      * @private {?Promise}
      */
     this.iframePromise_ = null;
-
+    /**
+     * @private {?Element}
+     */
+    this.placeholderWrapper_ = null;
```

Comment:
You're talking about `#firstLayoutCompleted`? We can override it to do nothing like https://github.com/ampproject/amphtml/blob/master/extensions/amp-anim/0.1/amp-anim.js#L66-L68